.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+go:
+  - 1.6
+  - tip
+
+before_install:
+  - go get github.com/Masterminds/glide
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/golang/lint/golint
```

Comment:
these packages will be useful for local development/testing as well - maybe we could "hide" them in Makefile (ex. `make install-test`)?